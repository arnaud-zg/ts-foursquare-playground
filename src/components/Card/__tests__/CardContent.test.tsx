import React from 'react'
import renderer from 'react-test-renderer'
import { CardContent } from '..'

describe('Component/CardContent', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<CardContent />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
