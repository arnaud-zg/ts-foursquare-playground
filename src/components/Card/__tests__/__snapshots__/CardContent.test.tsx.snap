// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component/CardContent renders correctly 1`] = `
<div
  className="pl-8 pr-8 pb-5 text-grey-darkest select-none"
/>
`;
